### DISCUSSION TITLE
Log panel: Colourise the entire line

### DISCUSSION THREAD

**Discussion by ollie-campbell** [grafana/grafana#74894] [General]
The log panel only highlights a specific logging level using a small bar on the left hand side of the panel.

There should be an option to highlight the entire line.

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move https://github.com/grafana/grafana/discussions/73424.

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!
